docker/common/install_ffmpeg.sh

```diff
@@ -94,4 +94,4 @@ ldconfig
 cd /
 rm -rf /tmp/ffmpeg*
 rm -rf /tmp/nv-codec-headers*
-rm -rf /var/lib/apt/lists/
```

Comment:
What's the reason for this?
Just because we're touching the common dockerfile, I'd want to be more cautious and let @thomasdhc also take a look

Reply:
I reverted this as part of the update to use `docker/Dockerfile` as-is then add the benchmark dependencies/entrypoint in a separate Dockerfile for benchmarking.